rust/noosphere/build.rs

```diff
@@ -0,0 +1,25 @@
+#![recursion_limit = "256"]
+use cfg_aliases::cfg_aliases;
+
+fn main() {
+    cfg_aliases! {
+        // Platforms
+        wasm: { target_arch = "wasm32" },
+        native: { not(target_arch = "wasm32") },
```

Comment:
I often put "native" in scare quotes when I write it because if you talk to someone who doesn't do web dev, often "native" is a meaningless term to them. Also, there is an argument to be made that "native" is relative and therefor JS/Wasm is "native" to web.

No action required, but just always find myself wondering if there is a better term to use here..

Reply:
`not-wasm` :upside_down_face: